traveltime-spec.yml

```diff
@@ -325,16 +344,16 @@ components:
           maxItems: 2000
           items:
             type: string
-        transportation:
-          $ref: '#/components/schemas/RequestTransportation'
-        travel_time:
-          $ref: '#/components/schemas/RequestTravelTime'
         departure_time:
           $ref: '#/components/schemas/RequestDepartureArrivalTime'
+        travel_time:
```

Comment:
General question - why the reorder? Harder to review because we have to check for actual changes.

Reply:
I reorder based on documentation structure